client/src/component/sos/audit/AuditReviewContainer.tsx

```diff
@@ -10,6 +11,10 @@ class AuditBallotListContainer extends React.Component<any, any> {
     public render() {
         const { history, sos } = this.props;
 
+        if (sos.asm.currentState === 'DOS_AUDIT_ONGOING') {
```

Comment:
Why is this conditional not in one place?

Reply:
This is because of how we are doing URL routing. I have an idea of a better place to put it, and indeed, to put other such ASM/state based redirects, but for the moment, this is simplest approach that does what we want.

Reply:
Ok.  Let's file an issue on that refactoring as a `someday` then.

Reply:
@kiniry filed #496.